using_images/s2i_images/dot_net_core.adoc

```diff
@@ -212,70 +167,6 @@ If you ran a standard installation, the image stream will be present.
 An image can be used to build an application by running `oc new-app` against a
 sample repository:
 
-ifdef::openshift-online[]
-----
-$ oc new-app dotnet:2.2~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.2 --context-dir=app
-$ oc new-app dotnet:2.1~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.1 --context-dir=app
-$ oc new-app dotnet:1.1~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-1.1 --context-dir=app
-$ oc new-app dotnet:1.0~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-1.0 --context-dir=app
-----
-endif::openshift-online[]
-
-ifndef::openshift-online[]
-----
-$ oc new-app registry.redhat.io/dotnet/dotnet-22-rhel7~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.2 --context-dir=app
-$ oc new-app registry.redhat.io/dotnet/dotnet-21-rhel7~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.1 --context-dir=app
-$ oc new-app registry.redhat.io/dotnet/dotnetcore-11-rhel7~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-1.1 --context-dir=app
-$ oc new-app registry.redhat.io/dotnet/dotnetcore-10-rhel7~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-1.0 --context-dir=app
-----
-endif::openshift-online[]
-
-ifdef::openshift-enterprise[]
-[NOTE]
-====
-The `oc new-app` command can detect .NET Core source starting in {product-title} 3.3.
-====
-endif::openshift-enterprise[]
-
-ifndef::openshift-online[]
-[[dot-net-core-templates]]
-== .NET Core Templates
-ifdef::openshift-enterprise[]
-[IMPORTANT]
-====
-The
-link:https://github.com/redhat-developer/s2i-dotnetcore/blob/master/templates[.NET
-image templates] and the .NET images streams must first be
-link:https://github.com/redhat-developer/s2i-dotnetcore#openshift-templates[installed].
-If you ran a standard installation, the templates and image streams will be
-present. This can be checked with:
-
-----
-$ (oc get -n openshift templates; oc get -n openshift is) | grep dotnet
-----
-====
-endif::openshift-enterprise[]
-{product-title} includes templates for the .NET Core images to help easily
-deploy a sample application.
-
-The link:https://github.com/redhat-developer/s2i-dotnetcore-ex[.NET Core sample
-application] running on `dotnet/dotnet-22-rhel7` can be deployed with:
-
-----
-$ oc new-app --template dotnet-example -p DOTNET_IMAGE_STREAM_TAG=dotnet:2.2 -p SOURCE_REPOSITORY_REF=dotnetcore-2.2
-----
-
-The link:https://github.com/redhat-developer/s2i-dotnetcore-ex[.NET Core sample
-application] running on `dotnet/dotnetcore-10-rhel7` can be deployed with:
-
-----
-$ oc new-app --template dotnet-example
-----
-
-The link:https://github.com/aspnet/MusicStore[.NET Core MusicStore application]
-using PostgreSQL as database can be deployed with:
-
 ----
-$ oc new-app --template=dotnet-pgsql-persistent
+$ oc new-app dotnet:3.1~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-3.1 --context-dir app
```

Comment:
This part needs QE to sign off on.

@tmds do you know who from QE can do that?

Reply:
_\*signs off*_